src/main/java/com/heatheryou/httpserver/constants/CharacterSet.java

```diff
@@ -0,0 +1,7 @@
+package com.heatheryou.httpserver.constants;
+
+public class CharacterSet {
+    public static final String EMPTY = "";
+    public static final String SP = " ";
```

Comment:
More descriptive to name it `SPACE` than just `SP`.

Reply:
I didn't think of it in this aspect and followed the way it was abbreviated in RFC docs, but it is definitely more descriptive to call it `SPACE`. I shouldn't have assumed that whoever would read my code would be familiar with the docs. Thank you for pointing this out!

Reply:
Fixed in 2d9c1b5f8ffbfaada5bef03095f5396ad5e3e28d